desk/app/groups.hoon

```diff
@@ -1810,6 +1877,24 @@
     =.  cor  ga-give-update
     ga-core
   ::
+  ++  ga-clone
+    ^+  ga-core
+    =/  =create:g
+      :*  name=q.flag
+          title=(crip "{<q.flag>} title")
+          description=(crip "{<q.flag>} description")
+          image=''
+          cover=''
+          *cordon:g
+          members=~
+          secret=%.n
+      ==
+    =.  cor
+      (emit %pass /groups/(scot %p our.bowl)/[q.flag]/create %agent [our.bowl %groups] %poke %group-create !>(create))
+    =.  cor
+      (emit %pass /groups/(scot %p p.flag)/[q.flag]/clone %agent [p.flag %groups] %watch /groups/(scot %p p.flag)/[q.flag]/clone)
```

Comment:
If we sent this to `our.bowl` instead of `p.flag` would that let us clone groups w/o installing this on the host? seems like it might work

Reply:
this does in fact work